OVHcloud Backint Agent with SAP HANA Cockpit (#5708)
ovh/docs: pages/hosted_private_cloud/sap_on_ovhcloud/cookbook_install_ovhcloud_backint_agent/guide.en-gb.md

/usr/sap/<SID>/HDB<NI>/exe/hdbsql -u SYSTEM -d SYSTEMDB "BACKUP DATA FOR <SID> USING BACKINT ('MANUAL_COMPLETE_BACKUP');"
```

You also have the possibility to trigger these backups via SAP HANA Cockpit. Select `Backint`{.action} in the `Destination Type`{.action} category.

![backup_hana_cockpit](images/backup_hana_cockpit.png){.thumbnail}

After the execution of these backups, several files named `_databackup_` are now present in your S3 Object Storage bucket, these files correspond to backups of your SAP HANA database via OVHcloud Backint Agent.

> Two files named `log_backup_0_0_0_0` have `DB_<SID>` and `SYSTEMDB` prefix are also present in your S3 Object Storage bucket. These files correspond to backups of the SAP HANA backup catalog, allowing you to list backups known by SAP HANA.

Reviewer: ```suggestion
Two files named `log_backup_0_0_0_0` and which have `DB_<SID>` and `SYSTEMDB` prefixes are also present in your S3 Object Storage bucket. These files correspond to backups of the SAP HANA backup catalog, allowing you to list backups known by SAP HANA.
```

We advise you to refer to the SAP Note [2782059](https://launchpad.support.sap.com/#/notes/2782059) which sets out four options to schedule the backup of SAP HANA databases.

> A daily scheduling of TENANTDB SAP HANA backup via SAP HANA Cockpit example:

Reviewer: ```suggestion
An example of a daily scheduling of a TENANTDB SAP HANA backup via the SAP HANA Cockpit:
```

### Recovery

To recover a SAP HANA database from a backup done with OVHcloud Backint Agent, follow these steps in SAP HANA Cockpit:

1\. In the `Database Management`{.action} tab, select your TENANTDB. Click on `Tenant Actions`{.action} then select `Recover Tenant`{.action}.

![recover_tenant_01](images/recover_tenant/recover_tenant_01.png){.thumbnail}

2\. The TENANTDB has to be stopped to do the recovery.

> Ensure to stop your SAP system linked to this SAP HANA database before starting the recovery.

Reviewer: ```suggestion
Make sure to stop your SAP system linked to this SAP HANA database before starting the recovery.
```

![recover_tenant_02](images/recover_tenant/recover_tenant_02.png){.thumbnail}

3\. Select the wanted option :

- `Data and logs backups`{.action} allows you to recover to a specific point in time.
- `Full data backup only`{.action} allows you to recover to a state of a full backup.

In our example, we select the `Data and logs backups`{.action} option.

![recover_tenant_03](images/recover_tenant/recover_tenant_03.png){.thumbnail}

> 4\. If you have previously select the `Data and logs backups`{.action} option, you have two options:

Reviewer: ```suggestion
4\. If you previously selected the `Data and logs backups`{.action} option, you have two options:
```

- `Recover to the most recent state`{.action} allows you to recover to the most recent state.
- `Recover to a specific point in time`{.action} allows you to recover to a specific point in time.

In our example, we select the `Recover to the most recent state`{.action} option.

![recover_tenant_04](images/recover_tenant/recover_tenant_04.png){.thumbnail}

> 5\. If you did not modify the backups catalog localisation, the `Backup location only`{.action} option is right.

Reviewer: ```suggestion
5\. If you did not modify the backups catalog location, the `Backup location only`{.action} option is right.
```

Otherwise, please give the path.

![recover_tenant_05](images/recover_tenant/recover_tenant_05.png){.thumbnail}

6\. After few seconds, all full data backups recorded in the backups catalog of your SAP HANA database are listed.

![recover_tenant_06](images/recover_tenant/recover_tenant_06.png){.thumbnail}

> 7\. You can leave the `Yes (recommended)`{.action} option select.

Reviewer: ```suggestion
7\. You can leave the `Yes (recommended)`{.action} option selected.
```

![recover_tenant_07](images/recover_tenant/recover_tenant_07.png){.thumbnail}

> 8\. If you did not modify the backups and logs backup localisation, please leave fields empty.

Reviewer: ```suggestion
8\. If you did not modify the backups and logs backup location, please leave the fields empty.
```

Otherwise, please set paths.

![recover_tenant_08](images/recover_tenant/recover_tenant_08.png){.thumbnail}

> 9\. We recommend selecting `Yes`{.action} for `Backint`{.action} category. This option allows to check the availability of backups before triggering the recover.

Reviewer: ```suggestion
9\. We recommend selecting `Yes`{.action} for the `Backint`{.action} category. This option allows you to check the availability of backups before triggering the recovery.
```

![recover_tenant_09](images/recover_tenant/recover_tenant_09.png){.thumbnail}

> 10\. In our example, we select `No`{.action} for `Initialize the log area`{.action} step.

Reviewer: ```suggestion
10\. In our example, we select `No`{.action} for the `Initialize the log area`{.action} step.
```

![recover_tenant_10](images/recover_tenant/recover_tenant_10.png){.thumbnail}

> 11\. Please check parameters of your recover before applying.

Reviewer: ```suggestion
11\. Please check parameters of your recovery before applying.
```

![recover_tenant_11](images/recover_tenant/recover_tenant_11.png){.thumbnail}

> 12\. The recover of your TENANTDB services starts.

Reviewer: ```suggestion
12\. The recovery of your TENANTDB services starts.
```
